devstack/lib/nfp

```diff
@@ -366,6 +354,7 @@ function launch_configuratorVM {
     GROUP="svc_management_ptg"
     echo "GroupName: $GROUP"
     PortId=$(gbp policy-target-create --policy-target-group $GROUP $InstanceName | grep port_id  | awk '{print $4}')
```

Comment:
Shouldn't the floating IP be created here instead of doing it in the external network create function?